requirements.txt

```diff
@@ -6,6 +6,8 @@ molo.yourwords==1.0.2
 molo.servicedirectory==2.1.2
 molo.polls==2.2.1
 molo.surveys==1.2.2
+wagtailmedia==0.1.1
+celery<4.0
```

Comment:
@saeedmb8 please take note - celery has a new version out a few days ago which breaks molo. You might start seeing failures cropping out.